public/app/features/chat/ChatIndex.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Page from 'app/core/components/Page/Page';
+import { useNavModel } from 'app/core/hooks/useNavModel';
+import { Chat } from '../../../../packages/grafana-runtime';
```

Comment:
simplifed after everything is moved to this folder

Reply:
done